arviz/data/datasets.py

```diff
@@ -199,7 +199,7 @@ def _sha256(path):
     return sha256hash.hexdigest()
 
 
-def load_arviz_data(dataset=None, data_home=None):
+def load_arviz_data(dataset=None, data_home=None, group_kwargs=None, regex=False):
```

Comment:
I would use `**kwargs` here and then document that they are passed to `from_netcdf` with intersphinx (see https://github.com/arviz-devs/arviz/issues/1188)

Reply:
done